src/styles/blocks/product.scss

```diff
@@ -0,0 +1,86 @@
+.product {
+  position: relative;
+  width: 200px;
+  height: 408px;
```

Comment:
No need to hardcode height - it should depend on content and margins/paddings between parts of content